cypher/refcard-tests/src/test/scala/org/neo4j/cypher/docgen/refcard/DatabaseManagementTest.scala

```diff
@@ -36,7 +36,7 @@ CREATE OR REPLACE DATABASE myDatabase
 ###assertion=update-two
 //
 
-CREATE IF NOT EXISTS myDatabase TOPOLOGY 1 PRIMARY 0 SECONDARIES
+CREATE OR REPLACE DATABASE myDatabase TOPOLOGY 1 PRIMARY 0 SECONDARIES
```

Comment:
and we are back with the `OR REPLACE` 🤣

Reply:
Pinged you about this on slack ;) (will be fixed in the next commit)